src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprintAction.java

```diff
@@ -96,20 +96,20 @@ public String getUrlName() {
     public @CheckForNull String getFingerprintHash(@CheckForNull String imageId) {
         return (imageId != null) ? DockerFingerprints.getFingerprintHash(imageId) : null;
     }
-    
+
```

Comment:
Please revert formatting-only changes, and configure your editor to not change whitespace in lines you have not otherwise edited.